src/js/api.js

```diff
@@ -119,6 +119,17 @@ export class API {
     return $.post({ url, data: userDetails });
   }
 
+  changePassword(endPoint, userDetails) {
+    // TODO: note that this won't work when other PR is merged due to "user" becoming "accessToken"
```

Comment:
Take extra care about this - probably best to merge the other PRs that changes it to `accessToken` into master, then merging master into this branch.

Reply:
This is resolved now - I merged the other PRs into master, then merged master into this branch, then changed `user` to `accessToken`